pkg/autohealing/healthcheck/healthcheck.go

```diff
@@ -76,6 +79,7 @@ func CheckNodes(checkers []HealthCheck, nodes []NodeInfo, controller NodeControl
 	for _, node := range nodes {
 		for _, checker := range checkers {
 			if !checker.Check(node, controller) {
+				node.FailedCheck = reflect.TypeOf(checker).String()
```

Comment:
I would rather we add an extra method(e.g. `GetName()`) in HealthCheck interface to get a meaningful checker type or name.

Reply:
OK, I will do that in next commit.